src/components/pages/CategoriesPage.js

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../../redux/categories/categories';
 
-const Categories = () => (<button type="button">Check status</button>);
+const Categories = () => {
+  const dispatch = useDispatch();
+  return (
+    <button
+      type="button"
+      onClick={() => dispatch(checkStatus())}
+    >
+      Check status
+    </button>
+  );
+};
```

Comment:
- [ ] Your implementation of the check status is almost done but there is one more work to be done. Kindly display the "User construction" message when the Check status button is clicked. It is supposed to fetch that message from the store and display it on the category page when the button is clicked. Please work on that and your project will be in a complete state 👍